e2e/test/scenarios/native-filters/sql-filters-source.cy.spec.js

```diff
@@ -585,7 +585,7 @@ describeEE("scenarios > filters > sql filters > values source", () => {
     cy.sandboxTable({
       table_id: PRODUCTS_ID,
       attribute_remappings: {
-        attr_uid: ["dimension", ["field", PRODUCTS.ID, null]],
+        attr_cat: ["dimension", ["field", PRODUCTS.CATEGORY, null]],
```

Comment:
This test was properly broken, as we were sandboxing on the wrong attribute.
It also is testing a filter that aggregates unique values and the request for that data does not include the `is_sandboxed` metadata.